lib/msal-common/src/network/INetworkModule.ts

```diff
@@ -31,7 +31,7 @@ export interface INetworkModule {
     sendGetRequestAsync<T>(
         url: string,
         options?: NetworkRequestOptions,
-        cancellationToken?: number
+        timeout?: number
```

Comment:
Can you please add a comment on the public API that is deprecated? 

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/23df98f6747f47f9ba1bb458feba9f183f4a902c/lib/msal-common/src/error/AuthError.ts#L18

Reply:
I'm not sure this is necessary. This is only changing the parameter name, and cancellationToken/timeout is only used internally.